app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,247 +11,172 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
-$sidebar-small-width: 75%;
-$sidebar-medium-width: 65%;
-$sidebar-large-width: 45%;
-$sidebar-footer-height: 5em;
-
-.darkswarm {
-  .shop-filters-sidebar {
-    display: flex;
-    flex-direction: column;
-    height: 100%;
-
-    .background {
-      position: fixed;
-      top: 0;
-      right: 0;
-      z-index: 200;
-      height: 100%;
-      width: 100%;
-      background-color: $shop-sidebar-overlay;
-      opacity: 0;
-      transition: opacity $transition-sidebar;
-    }
+.shop-filters-sidebar {
+  .property-selectors {
+    margin-bottom: $sidebar-footer-height + 2em;
+  }
+}
 
-    &.shown {
-      .background {
-        opacity: 1;
-      }
+products {
+  display: block;
 
-      .sidebar, .sidebar-footer {
-        margin-right: 0;
-      }
+  @include breakpoint(tablet) {
+    input.button.right {
+      float: left;
     }
+  }
 
-    .sidebar {
-      position: fixed;
-      top: 0;
-      right: 0;
-      z-index: 210;
-      height: 100%;
-      width: $sidebar-large-width;
-      margin-right: -$sidebar-large-width;
-      background-color: rgba($white, 0.95);
-      padding: 1em;
-      transition: margin $transition-sidebar;
-      overflow-y: scroll;
-
-      .property-selectors {
-        margin-bottom: $sidebar-footer-height + 2em;
-      }
+  @include breakpoint(mobile) {
+    .add_to_cart {
+      margin-top: 2rem;
     }
+  }
 
-    .sidebar-footer {
-      background-color: $grey-800;
-      width: $sidebar-large-width;
-      margin-right: -$sidebar-large-width;
-      height: $sidebar-footer-height;
-      position: fixed;
-      bottom: 0;
-      right: 0;
-      transition: margin $transition-sidebar;
-      padding: 1em;
-
-      button {
-        width: 48%;
+  form {
+    input.small.button.primary.right.add_to_cart {
+      &.dirty {
+        padding-left: 3.2rem;
       }
     }
 
-    @include breakpoint(tablet) {
-      .sidebar, .sidebar-footer {
-        width: $sidebar-medium-width;
-        margin-right: -$sidebar-medium-width;
-      }
-    }
+    i.cart-spinner {
+      position: absolute;
+      top: 14px;
+      right: 146px;
+      color: white;
+      font-size: 1.2em;
 
-    @include breakpoint(mobile) {
-      .sidebar, .sidebar-footer {
-        width: $sidebar-small-width;
-        margin-right: -$sidebar-small-width;
-      }
+      // Necessary to be below Z index of cart popover:
+      z-index: 98;
+      -webkit-animation: spin 2s infinite linear;
+      animation: spin 2s infinite linear;
     }
   }
 
-  products {
-    display: block;
+  product {
+    @include csstrans;
 
-    @include breakpoint(tablet) {
-      input.button.right {
-        float: left;
-      }
-    }
+    border-bottom: 1px solid #e5e5e5;
+    border-top: 1px solid #e5e5e5;
```

Comment:
Agree with CodeClimate. A new color deserves a new var, and then, we can question whether it is worth it or it's just 99% similar to an existing one.